### PR TITLE
Setup react replay (sdk v7) -  DO NOT MERGE THIS YET

### PR DESCRIPTION
**Pull request by realkosty**
based on https://github.com/sentry-demos/application-monitoring/pull/126

Tested https://sentry.io/organizations/testorg-az/issues/3146405066/events/f8fdfec1cad54db69b989ad627fcbda7/?project=5808623

### CODE REVIEW

**Comment by realkosty** [comment]
DO NOT MERGE THIS YET

**Review by thinkocapo — APPROVED** [review]

**Comment by realkosty** [comment]
Closed in favor of https://github.com/sentry-demos/application-monitoring/pull/157 (using new setup steps after replay was merged into @sentry/browser)

**Pull request closed without merging**
